code/script.js

```diff
@@ -29,19 +42,152 @@ const showMessage = (message, sender) => {
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
+// Function to push to next question
+const nextQuestion = (message) => {
+  if (questionNumber === 1) {
+    userAnswer(message)
+        nameInput.value = ""
+        setTimeout(() => reply1(message), 1000)
+  }
+
+  else if (questionNumber === 2) {
+  userAnswer(message)
+  nameInput.value = ""
+        setTimeout(() => reply2(message), 1000)
+  }
+
+  else if (questionNumber === 3) {
+    userAnswer(message)
+    nameInput.value = ""
+          setTimeout(() => reply3(message), 1000)
+  }
+
+  else if (questionNumber === 4) {
+      userAnswer(message)
+      nameInput.value = ""
+            setTimeout(() => reply4(message), 1000)
+  }
+  
+  else if (questionNumber === 5) {
+        userAnswer(message)
+        nameInput.value = ""
+              setTimeout(() => reply5(message), 1000)
+  }
+
+  else if (questionNumber === 6) {
+    userAnswer(message)
+    nameInput.value = ""
+          setTimeout(() => reply6(message), 1000)
+  }
+}
 
 // Starts here
+// When website loaded, chatbot asks first question.
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello gorgeous! What is your name?`, 'bot')
+}
+const reply1 = (message) => {
+  questionNumber++ 
+  botAnswer(`Hey ${message}, Which booking would you like to make? `)
+ 
+  inputWrapper.innerHTML = `
+     <button id="cutBtn" >Hair cut</button>
+     <button id="colorBtn" >Hair color</button>
+     <button id="spaBtn">Hair spa</button>`
+     
+   document.getElementById('cutBtn') .addEventListener('click', () => nextQuestion('Hair cut'))
+   document.getElementById('colorBtn').addEventListener('click', () => nextQuestion('Hair color'))
+   document.getElementById('spaBtn').addEventListener('click', () => nextQuestion('Hair spa'))
+ }
+ 
+ const reply2 = (message) => {
+   questionNumber ++ 
+   if (message === 'Hair cut') {
+    botAnswer(`Choose a Stylist below`)
+    inputWrapper.innerHTML = `
```

Comment:
Nice with the dropdown, haven't used it myself. Thanks for teaching me how to do it! 🥇